CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ This is the first candidate for the upcoming feature release v1.23.3 of Lotus. T
   - feat: make RunClientTest louder when deals fail ([filecoin-project/lotus#10817](https://github.com/filecoin-project/lotus/pull/10817))
   - fix: cli: Change arg wording in change-beneficiary cmd ([filecoin-project/lotus#10823](https://github.com/filecoin-project/lotus/pull/10823))
   - refactor: streamline error handling in CheckPendingMessages (#10818) ([filecoin-project/lotus#10818](https://github.com/filecoin-project/lotus/pull/10818))
+  - feat: Add tmp indices to events table while performing migration to V2
```

Comment:
Not relevant to this PR, but did we get rid of the contributors section?

Reply:
Yeah, partly because it was very corrupted by work in dependencies, and partly because GH has a new shiny way to view this that @jennijuju is excited about.